stdlib/http/src/main/java/org/ballerinalang/net/http/HttpConstants.java

```diff
@@ -138,6 +138,7 @@
     public static final String ANNOTATION_METHOD_OPTIONS = HTTP_METHOD_OPTIONS;
     public static final String ANN_NAME_PARAM_ORDER_CONFIG = "ParamOrderConfig";
     public static final String ANN_FIELD_PATH_PARAM_ORDER = "pathParamOrder";
+    public static final String IS_RESPONSE_PROCESSED = "isResponseProcessed";
```

Comment:
Shall we rephrase this as well?
```suggestion
    public static final String IS_DIRTY_RESPONSE = "isDirtyProcessed";
```